glossary: update baseline

PassionPenguin opened the pull request:
### Description

links modification

1. `baseline/index.md` wikipedia link removal
2. `baseline/typography/index.md` 
    1. wikipedia link name modified from `baseline` => `baseline (typography)`
    2. `chars of font` => `most chars of font`
    3. add east-asian baseline description
4. ~~`baseline/compatibility/index.md` wikipedia link added: `Baseline_(configuration_management)`~~

### Motivation

1. & 2.1 the wikipedia disambigious page got so many unrelated information that is unrelevent to the glossary (baseline*) we describe here.

    will it be better to link to baseline(typograph) and add a link to baseline(configuration management?), or just delete the link in baseline, add it respectively in baseline/compatibility & baseline/typography

2.2 some chars, like those have `descenders`(e.g. `y`), does not rest on it, but below the baseline its descender extends.

2.3 add description for east-asian chars baseline

### Additional details

![image](https://github.com/mdn/content/assets/41184947/54199fbd-e890-4f15-9681-2774174bcfdd)

[w: baseline_(configuration management)](https://en.wikipedia.org/wiki/Baseline_(configuration_management)) is related to baseline in Configuration, or in VCS, where it defines:

> a baseline is an agreed description of the attributes of a product, at a point in time, which serves as a basis for defining change.[[1]](https://en.wikipedia.org/wiki/Baseline_(configuration_management)#cite_note-1) A change is a movement from this baseline [state](https://en.wikipedia.org/wiki/State_(computer_science)) to a next state. The identification of significant changes from the baseline state is the central purpose of baseline identification.

[w: baseline_(typography)](https://en.wikipedia.org/wiki/Baseline_(typography)) is related to baseline in typography, where it defines:

> In European and West Asian [typography](https://en.wikipedia.org/wiki/Typography) and [penmanship](https://en.wikipedia.org/wiki/Penmanship), the baseline is the line upon which most letters sit and below which [descenders](https://en.wikipedia.org/wiki/Descender) extend.

### Related issues and pull requests

Fixes #33349 
Related PR #26676 where in the merge, baseline/compatibilitiy was added and baseline's link to wikipedia was modified to a disambigious link (`w: baseline_(typography)` => `w: baseline`).

PassionPenguin committed:
update links to wikipedia

wbamberg commented on files/en-us/glossary/baseline/compatibility/index.md:
```diff
@@ -61,3 +61,4 @@ If you have questions, feedback, or want to help update and expand the features
 - [web-platform-dx/web-features repository](https://github.com/web-platform-dx/web-features)
 - [W3C WebDX Community Group](https://www.w3.org/community/webdx/)
 - [mdn/browser-compat-data repository](https://github.com/mdn/browser-compat-data)
+- [Baseline (Configuration Management)](<https://en.wikipedia.org/wiki/Baseline_(configuration_management)>) on Wikipedia
```
I thought about this one but I'm not sure. Baseline in https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility is really a "marketing term", it's related to but not the same as baseline in configuration management.

PassionPenguin replied:
uhmm..i did think about this, as i first found the term `configuration baseline`;

> A set of specifications for a system, or Configuration Item (CI) within a system, that has been formally reviewed and agreed on at a given point in time, and which can be changed only through change control procedures. The baseline configuration is used as a basis for future builds, releases, and/or changes. [^1]

may u correct my understanding, that baseline table in mdn, is about "whether it's a feature that works in almost all, most, and few browsers", then it will match "the features can only be changed through change control procedures. used as a basis for future builds, releases and/or changes" (given that browser will and only will change the feature in future builds, releases and/or changes)..?

[^1]: [csrc.nist.gov](https://csrc.nist.gov/glossary/term/baseline_configuration)

PassionPenguin replied:
well....if not that matched, then should not the link `w: baseline` appears in `baseline/index.md`... as it's totally meaningless...(only one link in `w: baseline` points to both of the sub-entries's relevent information)

wbamberg replied:
> well....if not that matched, then should not the link `w: baseline` appears in `baseline/index.md`... as it's totally meaningless...(only one link in `w: baseline` points to both of the sub-entries's relevent information)

I agree that we should remove the link to https://en.wikipedia.org/wiki/Baseline in https://developer.mozilla.org/en-US/docs/Glossary/Baseline.

About the link to https://en.wikipedia.org/wiki/Baseline_(configuration_management) in https://pr33365.content.dev.mdn.mozit.cloud/en-US/docs/Glossary/Baseline/Compatibility, I'm 50/50. I do understand that they are related.

> it will match "the features can only be changed through change control procedures. used as a basis for future builds, releases and/or changes" (given that browser will and only will change the feature in future builds, releases and/or changes)..?

Not really. All browser features will only be changed "in future builds, releases and/or changes)". All browsers use change control for all features, whether they are "Baseline features" or not.

PassionPenguin replied:
emmm....yeap..

PassionPenguin committed:
link removal

PassionPenguin committed:
updates

1. which the characters rest => which **most of** the characters rest
2. add east-asian scripts description?

wbamberg reviewed (approved):
👍 nice, thank you @PassionPenguin !
